corehq/apps/hqwebapp/signals.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+
+from django.contrib.auth.signals import user_logged_in, user_login_failed
+from django.dispatch import receiver
+
+from corehq.apps.users.models import CouchUser
+
+
+def clear_login_attempts(user):
+    user = CouchUser.from_django_user(user)
```

Comment:
it seems a little strange that this function takes in a django user but then does everything with the couch user. Also, I noticed that we are inconsistent with `CouchUser.from_django_user(user)` versus `CouchUser.get_by_username(username)` (which is all the other one does). Maybe this function could just take a username (or a `CouchUser`)?